docs/pages/Configuration.md

```diff
@@ -702,6 +703,17 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.0
```

Comment:
Shouldn't the new `sourceMap` option also be documented in here? (added in https://github.com/SAP/ui5-project/pull/522)

Reply:
Not sure I quite understand this. 
Isn't the list below sufficient?